src/CMakeLists.txt

```diff
@@ -637,9 +637,15 @@ install(TARGETS ceph-mon DESTINATION bin)
 if(NOT ALLOCATOR STREQUAL "jemalloc")
   set(disable_jemalloc "DISABLE_JEMALLOC=1")
 endif()
+
+set(ROCKSDB_EXTRA_CXXFLAG -fPIC -Wno-unused-variable)
+if(NOT CMAKE_BUILD_TYPE STREQUAL Debug)
+  set(ROCKSDB_EXTRA_CXXFLAG ${ROCKSDB_EXTRA_CXXFLAG} -DNDEBUG)
+endif(NOT CMAKE_BUILD_TYPE MATCHES "Debug")
```

Comment:
On 27-7-2016 10:27, Kefu Chai wrote:
> In src/CMakeLists.txt
> <https://github.com/ceph/ceph/pull/10418#discussion_r72399688>:
> 
>> @@ -637,9 +637,15 @@ install(TARGETS ceph-mon DESTINATION bin)
>>  if(NOT ALLOCATOR STREQUAL "jemalloc")
>>    set(disable_jemalloc "DISABLE_JEMALLOC=1")
>>  endif()
>> +
>> +set(ROCKSDB_EXTRA_CXXFLAG -fPIC -Wno-unused-variable)
>> +if(NOT CMAKE_BUILD_TYPE STREQUAL Debug)
>> +  set(ROCKSDB_EXTRA_CXXFLAG ${ROCKSDB_EXTRA_CXXFLAG} -DNDEBUG)
>> +endif(NOT CMAKE_BUILD_TYPE MATCHES "Debug")
> 
> does not match with |if|.

Aaarrggh... Sorry about that.
I've got this root-canal infection, with splitting headache, for about 2
weeks now. And it is definitly impacting my concentration.

Will fix.